docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - db:/var/lib/postgresql/data
 
   app: &app_base
-    image: quay.io/foreman/foreman:develop
+    image: app/views/unattended/report_templates/ansible_inventory.erb #quay.io/foreman/foreman:develop
```

Comment:
This looks incorrect

Reply:
:man_facepalming: Thanks, it definitely doesn't belong here.